test/feedback/containers/Main.unit.spec.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import enzyme from 'enzyme';
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import { Main } from '../../../src/js/feedback/containers/Main';
+import DefaultView from '../../../src/js/feedback/components/DefaultView';
+import FeedbackForm from '../../../src/js/feedback/components/FeedbackForm';
+import FeedbackSubmitted from '../../../src/js/feedback/components/FeedbackSubmitted';
+
+const defaultProps = {
+  formIsVisible: false,
+  formValues: {},
+  formErrors: {},
+  setFormValues() {},
+  revealForm() {},
+  requestPending: false,
+  feedbackReceived: false,
+  shouldSendResponse: false,
+  sendFeedback() {},
+  clearError() {},
+  errorMessage: null
+};
+
+describe('<Main/>', () => {
+
+  it('should render with DefaultView', () => {
+    const wrapper = enzyme.shallow(<Main {...defaultProps}/>);
+    const text = wrapper.text();
+    expect(text.includes('Give feedback on this page')).to.be.true;
+    expect(wrapper.find(DefaultView)).to.have.lengthOf(1);
```

Comment:
Alternatively, you can do `wrapper.find('DefaultView')` as a string like that so you don't have to import it.